app/components/ReposList/tests/index.test.tsx

```diff
@@ -31,7 +32,7 @@ describe('<ReposList />', () => {
         error: false,
         loading: false,
         userData: {
-          repositories: false,
+          repos: false,
```

Comment:
we should either use `repos` everywhere, or `repositories` everywhere. I propose the shorter variant